recipes/r-saige/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+r_base:
+  - 3.5.1
+  - 3.6.1
```

Comment:
This doesn't support the most recent R version? As is this won't be installable in the same environment as any other packages produced in the past year.

Reply:
@dpryan79 yes and no. One of the dependencies for this version doesn't. I am starting recipe with this older version 0.35.8.8 - because it supports plain text dosage files. This is still used by some researchers. But the latest version 0.42.1 builds with R version 4, at least as far as I have tested locally

Reply:
for the record it's `r-spatest==3.0.0`, specific version is enforced by saige. Newer version of saige is using `r-spatest==3.1.2` which was build against R v4